src/tests/architect_tests/test_builders.py

```diff
@@ -254,10 +251,7 @@ def test_query_to_df():
         # create an engine and generate a table with fake feature data
         engine = create_engine(postgresql.url())
         create_schemas(
-            engine=engine,
-            features_tables=features_tables,
-            labels=labels,
-            states=states
+            engine=engine, features_tables=features_tables, labels=labels, states=states
```

Comment:
I know it's not the "Black" way, but I **strongly** prefer how this was before. It's the same reasoning as I believe Black gives for doing this with lists, *etc.* – it makes it *way* easier to understand diffs of invocations, (besides lists).

(It can also make it easier to read the invocation itself, outside of a diff.)

Reply:
I agree. It's possible a less opinionated tool like autopep8 would do better

Reply:
Though after further thinking about it, the effect may be muted given that it only applies to lines that fit in a reasonable line length. This line is probably towards the upper end of the length that will be formatted in this way, and I wouldn't mind reading a diff of that line.

Reply:
I still like the old way much better; but, yes, again, it's a question of what the automated tool does for this effort, and it won't do this for all such expressions 👍